docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md

```diff
@@ -19,6 +19,7 @@ To start feature integration, integrate the required features:
 |-|-|-|
 | Marketplace Product | dev-master | [Marketplace Product feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-product-feature-integration.html) |
 | Marketplace Merchant Portal Core | dev-master | [Marketplace Merchant Portal Core feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-merchant-portal-core-feature-integration.html) |
+| Marketplace Inventory Management  | dev-master | [Marketplace Inventory Management feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-inventory-management-feature-integration.html) |
```

Comment:
why? it is not listed in recommended dependencies: https://github.com/spryker-feature/marketplace-merchant-portal-product-management 
Please remove and declare all needed transfers in this IG

Reply:
In this PR here (https://github.com/spryker/spryker/pull/8421/files#diff-17928f854a728fb780dd37e1b663dee6573f39bacf8a1b7d67f3267bdf8a4cf4) we add a dependency from  the ProductMerchantPortalGui to the MerchantStock module which is part of the inventory-management-feature.

Reply:
I don't know why it was not added to the feature repository.

Reply:
ok, lets have it